docs/content/en/docs/getting-started/observability/_index.md

```diff
@@ -1,6 +1,244 @@
+to the observability data for your cluster.
+It is based on the
+[simplenode-dev](https://github.com/keptn-sandbox/klt-on-k3s-with-argocd)
+example.
+You can clone that repo to access it locally
+or just look at it for examples
+as you implement the functionality "from scratch"
+on your local Kubernetes deployment cluster.
+The
+[README](https://github.com/keptn-sandbox/klt-on-k3s-with-argocd/blob/main/setup/observability/README.md)
+file for that repo contains useful information.
+
+Two videos are available
+to walk you through this exercise if you prefer:
+
+* [Introducing Keptn Lifecycle Toolkit](https://youtu.be/449HAFYkUlY)
+* [Use SLOs and get DORA the Native K8s way!](https://www.youtube.com/watch?v=zeEC0475SOU)
+
+In the
+[Getting started with Keptn metrics](../metrics)
+exercise, you learn how to define and use Keptn metrics.
+You may want to complete that exercise before doing this exercise
+although that is not required.
+
+This exercise shows how to standardize access
+to the observability data for your cluster.
+The steps are:
+
+1. Install and configure the Lifecycle Toolkit on your cluster
+1. Configure OpenTelemetry for the Lifecycle Toolkit
+1. Create a Keptn application that aggregates your deployment workloads
+1. DORA metrics
+1. Using OpenTelemetry data
+1. Keptn metrics
+
+TODO: Make the list items links to the subsections.
+
+- Increment the version number for either your Workload
+  or your application to start aggragating data
+- View the aggregated metrics and traces as text.
+  You can also use Grafana or the dashboard of your choice
+  to view this information.
+- If you like, define `KeptnMetrics` for additional data you want to monitor.
+
+## Install and configure KLT
+
+The steps to install and configure KLT for this exercise are:
+
+* [Install KLT](#install-klt)
+* [Enable KLT on your cluster](#enable-klt-on-your-cluster)
+* [Annotate Deployment resource](#enable-klt-on-your-cluster)
+
+### Install KLT
+
+To install the Keptn Lifecycle Toolkit on your cluster:
+
+   - Be sure that your cluster includes the components discussed in
+     [Prepare your cluster for KLT](../../install/k8s.md/#prepare-your-cluster-for-klt)
+   - [Install and enable the Keptn Lifecycle Toolkit](../../install/install.md/#use-helm-chart)
+     on your cluster using the Helm chart
+
+If you installed KLT on your cluster for the
+[Getting started with Keptn metrics](../metrics)
+exercise, you do not need to re-install it for this exercise.
+However, if you only installed the `metrics-operator` for that exercise,
+you now need to install the full KLT.
+
+### Enable KLT on your cluster
+
+Follow the instructions in
+[Enable KLT for your cluster](../../install/install.md/#enable-klt-for-your-cluster)
+to enable KLT on your cluster.
+
+The
+[simplenode-dev-ns.yaml](https://github.com/keptn-sandbox/klt-on-k3s-with-argocd/blob/main/simplenode-dev/simplenode-dev-ns.yaml)
+file defines the `Namespace` resource for our example.
```

Comment:
I'm not sure I understand.  You don't have to enable KLT or apply annotations for metrics so yeah, the install section is a lot shorter.  I thought this was kind of parallel -- tell them the step, point to the regular docs for instructions.  And I had a link to the appropriate file in the sample which wasn't applicable for Metrics but I think is useful here.  We'll talk...

Reply:
@StackScribe this conversation is the only outstanding thing for me before a merge.

Reply:
I refactored the beginning material a bit and am waiting to hear if the new version satisfies @thisthat .  Long term, I want the landing page of the "Install" section to have this list and we'll just point to that from any Getting Started materials.

The first GS guide for KeptnMetrics is, of course, shorter and simpler because they don't need to enable KLT or annotate the yaml file.

Reply:
I agree to go ahead with the current version and do a follow-up to improve the overall install sections